src/libs/runtime-utils.js

```diff
@@ -10,8 +10,9 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
-export const DEFAULT_DISK_SIZE = 50
-export const DEFAULT_BOOT_DISK_SIZE = 50
+export const DEFAULT_DATAPROC_DISK_SIZE = 60 // For both main and worker machine disks. Dataproc clusters don't have persistent disks.
+export const DEFAULT_GCE_BOOT_DISK_SIZE = 70 // GCE boot disk size is not customizable by users. We use this for cost estimate calculations only.
```

Comment:
for cost estimation, we use the `bootDiskSize` returned from Leo...we shouldn't use the default defined in UI...or is this just a default if there's nothing returned form Leo?

Reply:
Yea we display cost estimates when user has no compute too so we need that info. It's not ideal in that we need to remember to change it if the default changes in Leo. The only way around that would be to get that info from a Leo API.